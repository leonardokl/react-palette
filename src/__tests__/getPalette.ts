import { getPalette } from "../";

test("return camelCased values", async () => {
  const actual = await getPalette("test");

  expect(actual).toMatchSnapshot();
});
