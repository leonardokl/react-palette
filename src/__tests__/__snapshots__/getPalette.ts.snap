// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`return camelCased values 1`] = `
Object {
  "darkMuted": "#2a324b",
  "darkVibrant": "#0e7a4b",
  "lightMuted": "#9cceb7",
  "lightVibrant": "#a4d4bc",
  "muted": "#64aa8a",
  "vibrant": "#b4d43c",
}
`;
